### PR TITLE
use sysadmin token only for add_read_acl

### PR DESCRIPTION
**Pull request by Tianhao-Gu**
addressing https://github.com/kbase/handle_service2/issues/92 and https://github.com/kbase/handle_service2/issues/93

### CODE REVIEW

**Comment by Tianhao-Gu** [comment]
@MrCreosote Could you also take a look at this PR? Thanks!

File: lib/AbstractHandle/Utils/Handler.py [hunk 1]
```diff
@@ -241,8 +241,8 @@ def add_read_acl(self, hids, token, username=None):
...
             node_id = handle.get('id')
             try:
-                self.shock_util.add_read_acl(node_id, token, username=username)
-            except Exception:
-                raise ValueError("Unable to set acl(s) on handles {}".format(handle.get('hid')))
-
+                self.shock_util.add_read_acl(node_id, username=username)
+            except Exception as e:
+                raise ValueError("Unable to set acl(s) on handles {}\n{}".format(handle.get('hid'),
+                                                                                 e))
```
Comment (MrCreosote) [inline lib/AbstractHandle/Utils/Handler.py:247]: Does this add the stacktrace into the message or just the exception type and message?

Regardless, I think you should add `from e` so the stacktrace for the primary exception is appended to the new exception.

**Review by MrCreosote — CHANGES_REQUESTED** [review]
Looks good, just one change

File: lib/AbstractHandle/Utils/Handler.py (hunk 1 above)
Comment (Tianhao-Gu) [reply lib/AbstractHandle/Utils/Handler.py]: 👍  cool. Thanks.

**Pull request merged**